user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -541,10 +542,17 @@ def __update_apps_with_prediction_info(self,
         try:
             # Build the QualCore handler object to handle the prediction model output
             q_core_handler = APIResultHandler().qual_core().with_path(qual_output_dir).build()
-            predictions_df = predict(platform=model_name, qual=qual_output_dir,
-                                     output_info=output_info,
-                                     model=estimation_model_args['customModelFile'],
-                                     qual_handlers=[q_core_handler])
+            if Utils.get_rapids_tools_env('QUALX_REVAMP'):
+                predictions_df = predict_x(platform=model_name,
+                                           qual=qual_output_dir,
+                                           output_info=output_info,
+                                           model=estimation_model_args['customModelFile'],
+                                           qual_handlers=[q_core_handler])
+            else:
+                predictions_df = predict(platform=model_name, qual=qual_output_dir,
+                                         output_info=output_info,
+                                         model=estimation_model_args['customModelFile'],
+                                         qual_handlers=[q_core_handler])
```

Comment:
We might want to eventually push the branching down into the core qualx APIs, just so all invocations to `predict()` can use the switch, but this is fine (and easier) for now.